db/migrations/20220819140008-add-dateRanges-to-bookingTypes-table.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+const async = require('async');
+
+var dbm;
+var type;
+var seed;
+
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+// The "status_timestamp" will capture the time when the reply's status is changed 
+// while the "updated" will capture the time when the reply's record is updated.
```

Comment:
Remove this comment as it is unrelated

Reply:
Fixed now, also added the feature to disable "add date range" button if there is only "always" date range already active